genai-perf/genai_perf/parser.py

```diff
@@ -1316,14 +1316,23 @@ def refine_args(
         args = _check_compare_args(parser, args)
     elif args.subcommand == Subcommand.TEMPLATE.value:
         pass
-    elif args.subcommand == Subcommand.PROCESS_EXPORT_FILES.to_cli_format():
+    elif args.subcommand == Subcommand.PROCESS.value:
         pass
     else:
         raise ValueError(f"Unknown subcommand: {args.subcommand}")
 
     return args
 
 
+def _set_output_config(config: ConfigCommand, args: argparse.Namespace) -> None:
```

Comment:
This method is redundant. These assignments are already done on lines 1439-1444 (of the original file) in the add_cli_options_to_config() method.